doc: documentation errors in cloud provider access (#1615)
mongodb/terraform-provider-mongodbatlas: website/docs/r/cloud_provider_access.markdown

* `last_updated_date`                - Date and time when this Azure Service Principal was last updated. This parameter expresses its value in the ISO 8601 timestamp format in UTC.
* `role_id`                        - Unique ID of this role.

> -> **NOTE:** Code examples of how the mongodbatlas_cloud_provider_access_setup resource and its attributes should be used to permit Atlas to access resources in your own cloud provider accounts are in Github, for [AWS](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples/atlas-cloud-provider-access/aws) and for [Azure](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples/atlas-cloud-provider-access/azure) 

Reviewer: ```suggestion
-> **NOTE:** For more details on how attributes are used to enable access to cloud provider accounts see [AWS example](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples/atlas-cloud-provider-access/aws) and [Azure example](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples/atlas-cloud-provider-access/azure). 
```
Small rephrasing to simplify.

> * -> **NOTE:** the two sub-attributes of the aws_config block will likely be needed on the external AWS account side as the constraints on the trust policy of an IAM role you're granting Atlas access to. They can be accessed via `mongodbatlas_cloud_provider_access_setup.<role>`, for example the trust policy will reference `mongodbatlas_cloud_provider_access_setup.<role>.aws_config[0].atlas_aws_account_arn` in its `Principal` statement, and the `mongodbatlas_cloud_provider_access_setup.<role>.aws_config[0].atlas_assumed_role_external_id` in its `Condition.StringEquals.sts:ExternalId` statement.

Reviewer 2: minor suggestions: apply formatting to `aws_config` plus adjust one run-on sentence.
```suggestion
* -> **NOTE:** the two sub-attributes of the `aws_config block` will likely be needed on the external AWS account side as the constraints on the trust policy of an IAM role you're granting Atlas access to. They can be accessed via `mongodbatlas_cloud_provider_access_setup.<role>`. For example, the trust policy will reference `mongodbatlas_cloud_provider_access_setup.<role>.aws_config[0].atlas_aws_account_arn` in its `Principal` statement, and the `mongodbatlas_cloud_provider_access_setup.<role>.aws_config[0].atlas_assumed_role_external_id` in its `Condition.StringEquals.sts:ExternalId` statement.
```

> * -> **NOTE:** the two sub-attributes of the aws_config block will likely be needed on the external AWS account side as the constraints on the trust policy of an IAM role you're granting Atlas access to. They can be accessed via `mongodbatlas_cloud_provider_access_setup.<role>`, for example the trust policy will reference `mongodbatlas_cloud_provider_access_setup.<role>.aws_config[0].atlas_aws_account_arn` in its `Principal` statement, and the `mongodbatlas_cloud_provider_access_setup.<role>.aws_config[0].atlas_assumed_role_external_id` in its `Condition.StringEquals.sts:ExternalId` statement.

Reviewer: We have an AWS example which helps understand the usage of both attributes: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/examples/atlas-cloud-provider-access/aws/aws-roles.tf#L19. Curious if you had seen this example, and if this note is worth adding having this defined.

Author: Ah, I didn't know about those examples. Is it worth putting a link to them in the documentation? Or is the link there already and I didn't see it?

Reviewer: Yes agree the examples are not very accessible from our docs. I believe we could add a link to the example I referenced within the `Example Usage with AWS` section, and remove this Note that was added. cc: @Zuhairahmed

Author: Got it - I've changed the note to a link
